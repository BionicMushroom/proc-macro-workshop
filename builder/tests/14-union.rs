use derive_builder::Builder;

#[derive(Builder)]
union Union {
    field0: u32,
    field1: u64,
}

fn main() {}
