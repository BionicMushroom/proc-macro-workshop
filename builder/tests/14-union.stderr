error: expected struct, found `union`
 --> tests/14-union.rs:4:1
  |
4 | union Union {
  | ^^^^^
